### DISCUSSION TITLE
Deklaration of proper names in Context Menu for unknown words

### DISCUSSION THREAD

**Discussion by NieLuc** [texstudio-org/texstudio#3334] [Ideas]
It would be nice if a option would be added to include a "add to dictionary as proper name" option in the context menu for unknown words. So the correction aid would know the word but wouldnt be seperated on a line break as that is often discouraged. If you want it to be seperated automatically you could just use the normal add to dictionary option.

**Comment by sunderme** [comment]
I don't understand.
Are you talking about hyphenation of text in converted latex documents ?

**Reply by NieLuc** [reply]
When I write a scientific paper, I often use company or product names, such as CANoe or Volkswagen. However, since these should not be separated at a line break (at least not in German), an option to mark them as proper nouns would be helpful, so that the word is included in the dictionary but does not have to be specially specified **_not_** to be separated. I'm sorry if something is unclear, English is not my mother tongue.

**Comment by sunderme** [comment]
the ambiguity is that latex handles text at two locations:
1) the source text, which is then compiled, is shown in texstudio and that is the only text texstudio directly handles with the dictionaries. 
2) after compilation, text is usually in the form of a pdf. There word breaks are relevant but that is handled by latex. Latex only takes information from the source text. So, avoiding hyphenation of a word is achieved by setting it in a command like `\mbox`.
The dictionaries of texstudio play no role here.
